ReadMe.md

```diff
@@ -8,101 +8,103 @@ The goal of this assignment is to build a simple REST API, implement tests for i
 
 ## Table of Contents
 1. [Overview](#overview)
-2. [Prerequisites](#prerequisites)
-3. [Project Setup](#project-setup)
-4. [REST API Development](#rest-api-development)
-5. [Testing and Coverage](#testing-and-coverage)
-    - [Unit Tests](#unit-tests)
+2. [Testing and Coverage](#testing-and-coverage)
     - [Integration Tests](#integration-tests)
     - [Code Coverage](#code-coverage)
-6. [API Testing](#api-testing)
-7. [Load Testing](#load-testing)
-8. [Reflection](#reflection)
-9. [Deliverables](#deliverables)
+3. [JMeter Load Testing](#load-testing)
+4. [Reflection](#reflection)
 
 ---
 
 ## Overview
-This project involves building a simple CRUD (Create, Read, Update, Delete) REST API, writing unit and integration tests for the API, ensuring at least 80% code coverage, and performing basic performance tests with load testing tools.
+This project involves building a simple CRUD (Create, Read, Update, Delete) REST API, writing integration tests for the API and trying to ensuring at least 80% code coverage.
 
----
-
-## Prerequisites
-Before beginning, ensure you have the following:
-- Programming language of choice: **Java**, **C#**, or other preferred languages.
-- Testing tools based on language:
-    - For **Java**: JUnit, Jacoco
-    - For **C#**: NUnit, Coverlet
-- **Postman** or **HTTP files** for API testing.
-- **JMeter**, **Artillery**, or **Gatling** for load testing.
-
-Ensure you have all dependencies and tools installed before starting.
+We didnt get to finish this project, because we used a lot of time on the basic setup of the project, and we had some issues with the testing of the API.
+Therefore we only managed to implement swagger, we managed to make integration tests for the API, we managed to look a little bit into the code coverage and we managed to look into JMeter.
 
 ---
 
-## Project Setup
-1. Clone this repository.
-2. Set up your development environment based on the language of choice.
-3. Install the necessary dependencies for unit testing, integration testing, and code coverage.
+## Testing and Coverage
+The things we managed to get done was:
+- Creating CRUD REST api, with possibility of testing via Swagger.
+- Creating integration tests for the API.
+- Ensuring (close to) 80% code coverage using Rider built-in test coverage.
+- Reflection
```

Comment:
Er det ikke det mere eller mindre det samme som der står oven over?

Reply:
Jo, Tror bare vi enten skal beholde linjen der forklarer hvad vi nåede eller listen ikke begge to